gpu-operator/release-notes.rst

```diff
@@ -69,6 +69,18 @@ New Features
   You can configure this in the Helm chart value by setting `dcgmexporter.service.internalTrafficPolicy` to `Local` or `Cluster` (default). 
   Choose Local if you want to route internal traffic within the node only.
 
+.. _v25.3.1-known-issues:
+
+Known Issues
+------------
+
+* For drivers 570.124.06, 570.133.20, and 570.148.08, 
```

Comment:
@klueska has suggested that we inform users of the upcoming R570 driver, which has a fix for this issue and is scheduled to be released in mid-July 2025. With this knowledge, users may opt to wait for that driver instead of going through multiple driver update cycles. 

Thoughts @chenopis @a-mccarthy  ?

Reply:
I don't think we should future promise in the docs, so probably best not to mention the upcoming R570 driver. I've added a Jira ticket in the backlog to remind me to update the docs once R570 has been released.